cpp/src/groupby/hash/groupby.cu

```diff
@@ -635,13 +623,12 @@ std::unique_ptr<table> groupby(table_view const& keys,
  * @brief Indicates if a set of aggregation requests can be satisfied with a
  * hash-based groupby implementation.
  *
- * @param keys The table of keys
  * @param requests The set of columns to aggregate and the aggregations to
  * perform
  * @return true A hash-based groupby should be used
  * @return false A hash-based groupby should not be used
  */
-bool can_use_hash_groupby(table_view const& keys, host_span<aggregation_request const> requests)
+bool can_use_hash_groupby(host_span<aggregation_request const> requests)
```

Comment:
This function also needs to be rewritten to remove structs check.

Reply:
No, it's still needed since it's checking values but not keys.

Reply:
But aren't we going to support new row operators for both keys and values?

Reply:
Looks like not, see #8039 and #10181

Reply:
So we are going to support nested types only in keys. The PR title may needs to be revised like "... to use new row operators for keys`.